src/app/frontend/deploy/deployfromsettings.scss

```diff
@@ -14,6 +14,19 @@
 
 @import '../variables';
 
+* {
+  font-size: 14px;
+}
+
+md-input-container {
+  margin-bottom: 5px;
```

Comment:
Base all margins on $baseline-grid. Make them natural multipliers of this value. Learn more at: https://www.google.com/design/spec/layout/responsive-ui.html#responsive-ui-grid